### DISCUSSION TITLE
Default Value in picklist

### DISCUSSION THREAD

**Discussion by crawe1** [YetiForceCompany/YetiForceCRM#14681] [General]
HI again :) @vovpff , @mariuszkrzaczkowski thank u for last help
i try to do picklist for IGDN module wich default filter. 

for example when create record in IGDN -> add inventory value -> open picklist with product. I want to add only for this module default value ">0" -> guys you have any ideas ?

i try to do with templetes but i dont know if this good idea becouse someone can change default filter

![image](https://user-images.githubusercontent.com/61950059/118559362-1d3dc000-b768-11eb-9b25-a3de688bdf32.png)

**Comment by vovpff** [comment]
No ideas. Filter way is more perfect. You can deny users ability to change filters.

Lets check modules/Products/Products.php for ability to replace All filter with ither one.

Another idea: for old yeti search modal content can be changed by modules/Products/Products.php. Maybe if you find way how to add conditions here.

**Reply by crawe1** [reply]
@vovpff 
Thank you but i think that this module can be bether tune becouse this table is for products in products module, but i think this table it should be for products in storage right. Becouse you issue products from dedicated storage. 

Maybe in future Yeti team do it new table to inventory name of products in storage :)

At this time i try your propouse.

**Reply by crawe1** [reply]
Hi @vovpff 
default filter not working by your idea when I add another value in advanced filter:
![image](https://user-images.githubusercontent.com/61950059/119465864-5ec00380-bd44-11eb-8ac6-d63c013da30a.png)
its this same effect if i not add this advanced value in filter. 
pick list is this same. 

i think that i must add some hardcode in tpl ? @vovpff  any more ideas ?
@mariuszkrzaczkowski  can you help ? How to grep value from this view picklist when qtyinstorage >0  module_name=IGDN view=edit

**Reply by vovpff** [reply]
Your idea to hardcode template looks crazy but well. I has no idea how to implement it but I think you need to add an additional php function to get list of products and use this function in template to render modal.
